Langs/0415.lang

```diff
@@ -1,11 +1,17 @@
 [MetaData]
 Language=Polish - Poland
 Translator=B.O.S.S (BxOxSxS)
-Version=2.3.0.0
+Version=2.3.0.3
+[Font]
+Extra Large=24
+Large=18
+Medium=10
+Small=8.5
 [Strings]
 Donate=Wesprzyj
 Check for Updates=Sprawdź aktualizacje
-Now Reach WhyNotWin11 via https://www.whynotwin11.org/=Teraz WhyNotWin11 dostępne na https://www.whynotwin11.org/
+Your Windows 11 Compatibility Results are Below=Twoje wyniki kompatybilności Windows 11 są poniżej
+Now Reach WhyNotWin11 via https://www.whynotwin11.org/=Teraz WhyNotWin11 jest dostępne na https://www.whynotwin11.org/
 Results Based on Currently Known Requirements!=Wyniki na podstawie obecnie znanych wymaganiach!
```

Comment:
Wyniki na podstawie obecnie znanych wymagań!
albo
Wyniki oparte na obecnie znanych wymaganiach!

Reply:
Faktycznie. Myślałem że to pierwsze jest niepoprawne a to jednak musi być dopełniacz